Improve tenancy doc (#1500)
kyma-incubator/compass: docs/director/03-tenancy.md

Director manages the configuration of the following main objects:

> * Application

Reviewer: ```suggestion
* [Application](https://github.com/kyma-incubator/compass/blob/master/docs/compass/02-01-components.md#application)
```

> * Runtime

Reviewer: ```suggestion
* [Runtime](https://github.com/kyma-incubator/compass/blob/master/docs/compass/02-01-components.md#kyma-runtime)
```

> * Label Definition

Reviewer: ```suggestion
* [LabelDefinition](https://github.com/kyma-incubator/compass/blob/master/docs/compass/03-02-labels.md#labeldefinitions)
```

> * Automatic Scenario Assigment

Reviewer: ```suggestion
* [Automatic Scenario Assigment](https://github.com/kyma-incubator/compass/blob/master/docs/compass/03-03-automatic-scenario-assignment.md)
```

> * Integration System

Reviewer: ```suggestion
* [Integration System](https://github.com/kyma-incubator/compass/blob/master/docs/compass/02-01-components.md#integration-system)
```

> * Application Template

Reviewer: ```suggestion
* [ApplicationTemplate](https://github.com/kyma-incubator/compass/blob/master/docs/compass/02-01-components.md#applicationtemplate)
```

> Applications, Runtimes, Label Definitions and Automatic Scenario Assigment and their child resources, such as APIs, are bound to tenants.

Reviewer: ```suggestion
Applications, Runtimes, LabelDefinitions, and Automatic Scenario Assignments and their child resources, such as APIs, are bound to tenants.
```

> Integration Systems and Application Templates are not bound to any tenant and can represent multiple tenants.

Reviewer: ```suggestion
Integration Systems and ApplicationTemplates are not bound to any tenant and can represent multiple tenants.
```
